src/main/java/backend/Season.java

```diff
@@ -1,8 +1,9 @@
 package backend;
 
 import com.google.gson.Gson;
+import com.google.gson.JsonSyntaxException;
 
```

Comment:
Auto import by IntelliJ. Besides that, I always prefer to import as little as absolutely possible into the local namespace. So I would prefer 2 imports above com.google.gson.* 
Another option would be to use gson.JsonSyntaxException but for Exceptions I prefer to import them into the local namespace since exceptions are very descriptive by themselves already.